src/common/classes/TempFile.cpp

```diff
@@ -229,7 +230,7 @@ void TempFile::init(const PathName& directory, const PathName& prefix)
 
 	if (doUnlink)
 	{
-		::unlink(filename.c_str());
+		::do_unlink(filename.c_str());
```

Comment:
On 09/02/2016 05:38 PM, Dimitry Sibiryakov wrote:
>> @@ -229,7 +230,7 @@ void TempFile::init(const PathName& directory, const PathName& prefix)
>>   
>>   	if (doUnlink)
>>   	{
>> -		::unlink(filename.c_str());
>> +		::do_unlink(filename.c_str());
> Temporary files on Windows are created with "delete on close" flag and never deleted explicitly. So, erasing of files don't, actually, works there.
>

On posix things are looking more or less same - file is unlinked right 
after open, after it file can't be accessed from OS but descriptor 
remains opened in a process, can be used (and is used) before close() 
call. Looks like for temporary files wipe should be also done on close 
for all platforms.